dipy/align/imaffine.py

```diff
@@ -903,7 +903,7 @@ def __init__(
         options=None,
         verbosity=VerbosityLevels.STATUS,
     ):
-        """Initialize an instance of the AffineRegistration class.
+        r"""Initialize an instance of the AffineRegistration class.
```

Comment:
I do not understand why you need a raw docstring here. Can you explain ?

Reply:
My bad. I may have added it thinking that a warning in this method was raised by the math expressions. Removed in 1b8fea5.